src/main/java/org/broadleafcommerce/menu/processor/AbstractMenuProcessorExtensionHandler.java

```diff
@@ -30,7 +30,7 @@ public abstract class AbstractMenuProcessorExtensionHandler extends AbstractExte
         implements MenuProcessorExtensionHandler {
 
     @Override
-    public ExtensionResultStatusType addAdditionalFieldsToModel(String tagName, Map<String, String> tagAttributes, Map<String, Object> newModelVars, BroadleafTemplateContext context) {
+    public ExtensionResultStatusType addAdditionalFieldsToModel(Map<String, Object> newModelVars, String menuId, String menuName) {
```

Comment:
this is just a stub, there is a real implementation in Theme module I don't see PR for the theme module.
I don't see why would it need menu id as current impl is interested in menuName only.